src/main.py

```diff
@@ -1,7 +1,343 @@
 import os
 import boto3
 from botocore.exceptions import ClientError
+from pdf_parser import Pdf_Parser
 
+def get_pdf_requirements_validation(pdf):
+
+    parser = Pdf_Parser(pdf) # this takes in a file name??
```

Comment:
@themnsavage I'm not entirely sure what the parameter pdf here is. The Pdf_parser object appears to take in a file path and extract information from the PDF in that file path. So I'm not sure what the variable pdf is supposed to be here or how to translate that information to the Pdf_parser object

Reply:
Assume that it is taking the file name. Because when you decode the base64 pdf it will return a I/O that mimics a file name.

Reply:
Ok, thanks. Then I think this PR is ready for a review